esp-lp-hal/src/i2c.rs

```diff
@@ -13,7 +13,7 @@ const I2C_LL_INTR_MASK: u32 = (1 << LP_I2C_TRANS_COMPLETE_INT_ST_S)
 const LP_I2C_FIFO_LEN: u32 = 16;
 
 #[doc(hidden)]
-pub unsafe fn conjour() -> Option<LpI2c> {
+pub unsafe fn conjure() -> Option<LpI2c> {
```

Comment:
Why is this an `Option`?

Reply:
TBH I just copied it from GPIO driver - but doesn't make sense to have Option for UART and I2C, removed, thanks!